Annotato/Annotato/Views/Document/Edit/DocumentEditViewController.swift

```diff
@@ -70,10 +70,21 @@ class DocumentEditViewController: UIViewController, AlertPresentable, SpinnerPre
         documentView.leftAnchor.constraint(equalTo: margins.leftAnchor).isActive = true
         documentView.rightAnchor.constraint(equalTo: margins.rightAnchor).isActive = true
     }
+
+    private func saveDocument() {
+        Task {
+            guard let documentViewModel = documentViewModel else {
+                return
+            }
+
+            self.documentViewModel = await DocumentController.updateDocument(document: documentViewModel)
```

Comment:
I'm not sure if the subviews are updated (to bind to the new view model) when the view model switches. For example the annotation views would still be bound to the old annotation view models?

Reply:
You're right, I'll remove the assignment to prevent confusion then.

Reply:
I think that works for now, since we goBack immediately, we will need to refresh views in future